stdlib/3/pyclbr.pyi

```diff
@@ -0,0 +1,40 @@
+from typing import List, Union, Sequence, Optional, Dict
+
+
+class Class:
+    module = ...  # type: str
+    name = ...  # type: str
+    super = Optional[List[Union['Class', str]]]
```

Comment:
Please use `"` to quote types, to be consistent with (most of) the other files in typeshed.